Dockerfile

```diff
@@ -1,23 +1,30 @@
-FROM node:14.18-stretch
+FROM mhart/alpine-node:14.17.3
```

Comment:
why not use the official node alpine images? https://hub.docker.com/_/node

Reply:
good callout, updated and confirmed working with the official distro